virtual/docker/docker_test.go

```diff
@@ -37,8 +37,9 @@ func testCleanup(t *testing.T, c hkndocker.Container) func() {
 }
 
 func TestContainerBase(t *testing.T) {
-	// testing create
+	// testing create. Do a long sleep to keep it alive
 	c1 := hkndocker.NewContainer(hkndocker.ContainerConfig{
+		Cmd:   []string{"sleep", "1d"},
 		Image: "alpine",
 	})
```

Comment:
Having a long sleep in CI might cause problems, I think, for the sake of tests, it could be skipped during CI test or sleep amount could be much less than 1 day.

Reply:
Sorry for the late comment. 

It is not meant the run for one day, pretty sure stopping is part of the test.

Reply:
Ah, ok then. I thought it is one day.